fem/libceed/ceed-wrappers.hpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_CEED_WRAPPERS_HPP
+#define MFEM_CEED_WRAPPERS_HPP
+
+#include "../../config/config.hpp"
+
+#ifdef MFEM_USE_CEED
+#include <ceed.h>
+#include "ceedsolvers-interpolation.h"
+#include "../../linalg/operator.hpp"
+
+namespace mfem
+{
+
+/// Wraps a CeedOperator in an mfem::Operator
+class MFEMCeedOperator : public Operator
```

Comment:
Do you want me to do some of this here before merging or wait for #1924 ?

Reply:
To clarify a bit, I was thinking that this would be renamed `ConstrainedCeedOperator` and inherited from a `ConstrainedOperator` and that the constructor would use an `UnconstrainedCeedOperator`.